dual_net.py

```diff
@@ -33,7 +33,7 @@
 import go
 
 EXAMPLES_PER_GENERATION = 2000000
-TRAIN_BATCH_SIZE = 16
+TRAIN_BATCH_SIZE = 256
```

Comment:
Can we add some comments where why we chose these values?

Reply:
Yep i can add some comments.  really though, these are getting out of hand and need to be consolidated somehow.